Classes/common/CDTIncrementalStore.m

```diff
@@ -2208,6 +2221,75 @@ - (id)executeSaveRequest:(NSSaveChangesRequest *)saveRequest
     return @[];
 }
 
+- (id)executeBatchUpdateRequest:(NSBatchUpdateRequest *)updateRequest
```

Comment:
It looks like you always return a `CDTISBatchUpdateResult`, so lets be specific about that, rather than use `id`